dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -54,8 +97,44 @@ def _CreateHunt(
     """
     runner_args = self.grr_api.types.CreateHuntRunnerArgs()
     runner_args.description = self.reason
+
+    if self.match_mode:
+      if self.match_mode.lower() == 'any':
+        match_mode = runner_args.client_rule_set.MATCH_ANY
+      elif self.match_mode.lower() == 'all':
+        match_mode = runner_args.client_rule_set.MATCH_ALL
+      else:
+        self.ModuleError(f'Unknown match mode {self.match_mode}', critical=True)
```

Comment:
You're already making sure that match mode is good in HuntSetup(), no? If so, no need to check here.

Reply:
Yes that's correct.  Removed the check.